src/main/resources/templates/aeolus/java/plain_maven_blackbox.sh

```
@@ -6,36 +6,48 @@
+  echo '⚙️ executing custom_checkers'
+  # all java files in the assignment folder should have maximal line length 80
+  pipeline-helper line-length -l 80 -s ${studentParentWorkingDirectoryName}/ -e java
+  # checks that the file exists and is not empty for non gui programs
+  pipeline-helper file-exists ${studentParentWorkingDirectoryName}/Tests.txt
+}
+
+replace_script_variables () {
+  echo '⚙️ executing replace_script_variables'
+  local JAVA_FLAGS="-Djdk.console=java.base"
+  local testfiles_base_path="./testsuite/testfiles"
+  local tool=$(find testsuite -name "*.tests" -type d -printf "%f" | sed 's#.tests$##')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)